doc/user_manual/sampler.tex

```diff
@@ -3285,3 +3294,130 @@ \subsubsection{Metropolis (Metropolis-Hastings Sampler)}
   ...
 </Samplers>
 \end{lstlisting}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%% Adaptive Metropolis Sampler %%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsubsection{Adaptive Metropolis Sampler}
+\label{subsubsubsec:adaptiveMetropolis}
+
+The search for improved proposal distributions of Metropolis sampler is often
+done manually, through trial and error, though this can be difficult especially
+in high dimensions. An alternative approach is adaptive Metropolis, which asks
+the computer to automatically "learn" better parameter values "on the fly".
+
+\specBlock{an}{AdaptiveMetropolis}
+%
+\attrIntro
+
+\begin{itemize}
+  \itemsep0em
+  \item \nameDescription
+\end{itemize}
+
+\variableIntro{AdaptiveMetropolis}
+
+\begin{itemize}
+  \item \variableDescription
+    \variableChildrenIntro
+    \begin{itemize}
+      \item \distributionDescription
+      \item \functionDescription
+      \item \xmlNode{initial}, \xmlDesc{float, optional field}, specified the initial value for given variable.
+      \item \xmlNode{proposal}, \xmlDesc{Assembler Object}, specifies the proposal distribution for this variable.
+      \nb We only allow one-dimensional symmetric distribution to be the proposal distribution.
+      This node must contain the following two attributes:
+      \begin{itemize}
+        \item \xmlAttr{class}, \xmlDesc{required string attribute}, the main
+          ``class'' of the listed object. Only ``Distributions'' is allowed.
+        \item \xmlAttr{type},  \xmlDesc{required string attribute}, the object
+          identifier or sub-type.
+        \item \xmlAttr{dim}, \xmlDesc{positive integer, optional attribute}, required for
```

Comment:
The proposal can  be multivariate or there is the same limitation in the ``not adaptive''?